File: app/widgets/ncbi/seqs/action_panel.js
//= require <templates/action_panel>

JooseModule('Ncbi.Seqs', function () {
  JooseClass('ActionPanel', {
    isa: Templates.ActionPanel,
    has: {
      context: { is: 'ro', required: true, nullable: false },
      buttons: { is: 'ro', init: function () { return(
            [ { label: 'Import',
                img: { src: '/images/addnew.gif' }
              }
            ]
          )
      }}
    }
  })
});

File: app/widgets/molecular/insd/seqs/seq_catalogs/action_panel.rb
class Molecular::Insd::Seqs::SeqCatalogs::ActionPanel < Templates::ActionPanel
  def initialize options
    options = { buttons: [ { label: 'Add to Cell', img: { src: '/images/small_addnew.gif' } } ] }.merge(options)
    super
    if interact_mode != 'edit'
      @buttons = {}
    end
  end
end